testcases/OpTestEEH.py

```diff
@@ -82,8 +93,30 @@ def gather_logs(self):
         c.run_command(cmd)
         c.run_command("dmesg")
 
+    def check_eeh_hit(self):
+        c = self.cv_SYSTEM.sys_get_ipmi_console()
+        tries = 10
+        for i in range(1, tries+1):
+            res = c.run_command("dmesg | grep 'EEH: Frozen';echo $?")
+            if int(res[-1]) == 0:
+                return True
+            time.sleep(1)
+        else:
+            return False
+
+    def check_eeh_removed(self):
+        tries = 30
+        c = self.cv_SYSTEM.sys_get_ipmi_console()
+        for i in range(1, tries+1):
+            res = c.run_command("dmesg | grep 'permanently disabled'; echo $?")
```

Comment:
Yes, we are seeing this message for all different kinds of adapter, and we see this message consistently.